app/controllers/org_admin/phases_controller.rb

```diff
@@ -26,12 +26,9 @@ def show
     def edit
       phase = Phase.includes(:template).find(params[:id])
       authorize phase
-      if !phase.template.latest?
-        flash[:notice] = _('You are viewing a historical version of this template. You will not be able to make changes.')
-      end
       section = params.fetch(:section, nil)
       # User cannot edit a phase if its a customization so redirect to show
-      if phase.template.customization_of.present?
+      if phase.template.customization_of.present? || !phase.template.latest?
```

Comment:
where are we handling the case of editing a 'own' phase created in a template that is customised?